src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+            printTask(Tasks[taskCounter], taskCounter);
+        } catch(Exception e) {
+            System.out.println("    Error: Insufficient detail");
+        }
+    }
+
+    private static void addTodo(Task[] Tasks, int taskCounter, String[] userCommands) {
+        try {
+            Tasks[taskCounter] = new Todo(userCommands[1]);
+            printTask(Tasks[taskCounter], taskCounter);
+        } catch(Exception e) {
+            System.out.println("    Error: Insufficient detail");
+        }
+    }
+
+    private static void printDone(String[] input, Task[] Tasks) {
+        try {
+            int num = Integer.parseInt(input[1]) - 1;
+            Tasks[num].taskDone();
+            System.out.println("    Nice! I've marked this task as done:");
+            System.out.println("      " + Tasks[num]);
+        }catch(NullPointerException e) {
+            System.out.println("    Error: Given Index is out of bound");
+        }catch(Exception e) {
+            System.out.println("    Error: Insufficient detail");
+        }
+    }
+
+    private static void printTask(Task task, int taskCounter) {
+        System.out.println("    Got it! I've added this task:");
+        System.out.println("       " + task);
+        taskCounter ++;
+        System.out.println("    Now you have " + taskCounter + " task(s) in the list.");
+    }
+
+    private static void printList(Task[] tasks, int taskCounter) {
+        System.out.println("    Here are the tasks in your list:");
+        for (int i = 0; i < taskCounter; i++) {
+            System.out.println("    " + (i + 1) + "." + tasks[i]);
+        }
+    }
+
+    private static void printByeMessage() {
+        System.out.println("    Bye. Hope to see you soon!");
+        System.out.println(DIVIDER);
+    }
+
+    private static String getInput(String next) {
+        String input = next;
+        System.out.println(DIVIDER);
+        return input;
+    }
+
+    private static void printWelcomeMessage() {
+        String logo = "    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░\n" +
+                "    ░░░░░░░░░░▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄░░░░░░░░░\n" +
+                "    ░░░░░░░░▄▀░░░░░░░░░░░░▄░░░░░░░▀▄░░░░░░░\n" +
+                "    ░░░░░░░░█░░▄░░░░▄░░░░░░░░░░░░░░█░░░░░░░\n" +
+                "    ░░░░░░░░█░░░░░░░░░░░░▄█▄▄░░▄░░░█░▄▄▄░░░\n" +
+                "    ░▄▄▄▄▄░░█░░░░░░▀░░░░▀█░░▀▄░░░░░█▀▀░██░░\n" +
+                "    ░██▄▀██▄█░░░▄░░░░░░░██░░░░▀▀▀▀▀░░░░██░░\n" +
+                "    ░░▀██▄▀██░░░░░░░░▀░██▀░░░░░░░░░░░░░▀██░\n" +
+                "    ░░░░▀████░▀░░░░▄░░░██░░░▄█░░░░▄░▄█░░██░\n" +
+                "    ░░░░░░░▀█░░░░▄░░░░░██░░░░▄░░░▄░░▄░░░██░\n" +
+                "    ░░░░░░░▄█▄░░░░░░░░░░░▀▄░░▀▀▀▀▀▀▀▀░░▄▀░░\n" +
+                "    ░░░░░░█▀▀█████████▀▀▀▀████████████▀░░░░\n" +
+                "    ░░░░░░████▀░░███▀░░░░░░▀███░░▀██▀░░░░░░\n" +
+                "    ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░\n";
```

Comment:
Nice use of refactoring and interesting mascot used!

Reply:
👍